### ISSUE TITLE
IE 11 bug

### ISSUE THREAD

**Issue by Daniel-Glucksman** [grofit/treacherous-aurelia#9]
Hi

There seem to be several bugs with IE 11. First off when a property is marked invalid the label stays and won't ever go back to valid. 

There are also errors in the console which I assume is related to this 
Object doesn't support property or method 'remove'

I beleive it comes from below which is located in InlineStrategy

```
 if (errorElement) {
                errorElement.remove();
```

**Comment by grofit** [comment]
thats a bit rubbish, I guess the more compatible way would be to get the parent node then tell it to remove child which would be the error element.

**Comment by grofit** [comment]
This should be fixed in 0.2.1 feel free to re-open if needed.
